lobbyfacts/data/extract/meetingmaps.py

```diff
@@ -86,6 +90,8 @@
           (u'Director-General Walter Radermacher', u'http://ec.europa.eu/transparencyinitiative/meetings/meeting.do?host=b85b3c8d-483b-4e3d-b066-160c467e2884'),
           (u'Director-General Xavier Prats Monn\xe9', u'http://ec.europa.eu/transparencyinitiative/meetings/meeting.do?host=9bd9f7c0-836e-4ff6-abf4-a1f8650861cb'),
           (u'Director-General Ann Mettler', u'http://ec.europa.eu/transparencyinitiative/meetings/meeting.do?host=0d0d638f-322e-44fc-8c9c-875438c4a719'),
+          (u'', u'http://ec.europa.eu/transparencyinitiative/meetings/meeting.do?host=0d0d638f-322e-44fc-8c9c-875438c4a719'),
```

Comment:
the url above fore Ann Mettler seems exactly the same, what is the context here?